pkgs/development/libraries/webkitgtk/default.nix

```diff
@@ -222,6 +230,17 @@ stdenv.mkDerivation rec {
     sed 43i'#include <CommonCrypto/CommonCryptor.h>' -i Source/WTF/wtf/RandomDevice.cpp
   '';
 
+  postInstall = ''
+    mv $out/libexec/MiniBrowser $out/bin/MiniBrowser
+    gappsWrapperArgsHook # FIXME: currently runs at preFixup
+    wrapGApp $out/bin/MiniBrowser --argv0 MiniBrowser
```

Comment:
Why not do this in `postFixup` then?

Reply:
Also could you add comment about why are we wrapping this? (Missing GSettings schemas? GStreamer plug-ins?)